protocol/contracts/libraries/Silo/LibGerminate.sol

```diff
@@ -129,10 +145,14 @@ library LibGerminate {
                 -int256(totalGerm.deposited[tokens[i]].amount),
                 -int256(totalGerm.deposited[tokens[i]].bdv)
             );
-
             // clear deposited values.
             delete totalGerm.deposited[tokens[i]];
         }
+
+        // emit change in total germinating stalk.
+        // safecast not needed as finishedGerminatingStalk is initially a uint128.
+        emit TotalGerminatingStalkChanged(season, -int256(finishedGerminatingStalk));
```

Comment:
shouldn't this be `germinationSeason`?

Reply:
fixed here: https://github.com/BeanstalkFarms/Beanstalk/pull/837/commits/cdc5a0353e9298e5d74888fde498d3a9988114cc#diff-d2261445685829aad9421eae03f9644ae9d11be5713ccc0870497a0be1e58d8aR154